elements-harness/src/image.rs

```diff
@@ -143,7 +144,7 @@ impl IntoIterator for ElementsCoreImageArgs {
 
         match self.network {
             Network::Testnet => args.push("-testnet".to_string()),
-            Network::Regtest => args.push("-regtest".to_string()),
```

Comment:
I honestly don't know what the difference between `regtest` and a custom chain is. I followed the instructions from here: https://docs.blockstream.com/liquid/node_setup.html